test/ui/widgets/grade_button_test.dart

```diff
@@ -55,6 +55,7 @@ void main() {
   group("GradeButton -", () {
     setUp(() async {
       intl = await setupAppIntl();
+      setupNavigationServiceMock();
```

Comment:
Can you add a test to see validate the behaviour of tapping a button? Go see the BottomBar test for an example.

Reply:
@apomalyn have a look and tell me if this is what you wanted?

Reply:
@emondhar I think the idea was to test if we get properly redirected to the grades page when tapping on a grade button that is present in the grades card.

Reply:
ahh ok, I get it now.

Reply:
What about now?

Reply:
@elornHamon